### PR TITLE
Fixsqwarn

### PR DESCRIPTION
**Pull request by minborg**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/bytes/MappedFile.java [hunk 1]
```diff
@@ -283,6 +283,7 @@ public void setNewChunkListener(final NewChunkListener listener) {
     @NotNull
     public abstract RandomAccessFile raf();
 
+    @Deprecated
```
Comment (alamar) [inline src/main/java/net/openhft/chronicle/bytes/MappedFile.java:291]: Should we add comment "To be removed in 2.2n"?

File: src/main/java/net/openhft/chronicle/bytes/internal/ChunkedMappedFile.java [hunk 2]
```diff
@@ -358,6 +358,7 @@ public RandomAccessFile raf() {
         return raf;
     }
 
+    @Deprecated
```
Comment (alamar) [inline src/main/java/net/openhft/chronicle/bytes/internal/ChunkedMappedFile.java:366]: ditto

**Review by alamar — CHANGES_REQUESTED** [review]

File: src/main/java/net/openhft/chronicle/bytes/MappedFile.java (hunk 1 above)
Comment (minborg) [reply src/main/java/net/openhft/chronicle/bytes/MappedFile.java:291]: The removal of the override is a bit complicated. We need to move from `finalize()` to perhaps using `WeakReference` or something else. The end-user should never invoke this method so I do not think we have to say when.

Comment (minborg) [reply src/main/java/net/openhft/chronicle/bytes/MappedFile.java:291]: In other words, it is not a part of the public API.

**Review by alamar — APPROVED** [review]

**Pull request merged**